numpy/core/fromnumeric.py

```diff
@@ -3562,8 +3562,8 @@ def var(a, axis=None, dtype=None, out=None, ddof=0, keepdims=np._NoValue):
 
     See Also
     --------
-    std, mean, nanmean, nanstd, nanvar
-    ufuncs-output-type
+    numpy.std, numpy.mean, numpy.nanmean, numpy.nanstd, numpy.nanvar
```

Comment:
Here again, I assume the purpose is to get the links in `np.ma.var` to work correct? Note that there is already a `np.ma.std` and `np.ma.mean`, so those two links work appropriately (and target the ma versions of those functions, which is preferable).

However there aren't `ma` versions of the `nan*` functions, perhaps the approach works for those.